src/Analysis/GWASUIApp/GWASJob.jsx

```diff
@@ -0,0 +1,177 @@
+import React, { useEffect } from 'react';
+import {
+  Button, List, Tag, Popconfirm,
+} from 'antd';
+import {
+  CheckCircleOutlined,
+  SyncOutlined,
+  CloseCircleOutlined,
+  QuestionCircleOutlined,
+} from '@ant-design/icons';
+import { useQuery, useMutation } from 'react-query';
+import PropTypes from 'prop-types';
+import { gwasWorkflowPath } from '../../localconf';
+import { headers, fetchAndSetCsrfToken } from '../../configs';
+import { getPresignedUrl } from '../AnalysisJob';
+
+const GWASJob = ({ workflow, refreshWorkflows }) => {
+  useEffect(() => {
+    fetchAndSetCsrfToken().catch((err) => { console.log('error on csrf load - should still be ok', err); });
+  }, []);
+
+  async function handleWorkflowOutput(url) {
+    const response = await fetch(url, { headers }).then((res) => res.json()).then((data) => data);
+    if (response) {
+      getPresignedUrl(JSON.parse(response.outputs.parameters[0].value).did, 'download')
+        .then((res) => {
+          window.open(res, '_blank');
+        });
+    }
+  }
+
+  const getActionButtons = (phase, workflowName) => {
+    const actionButtons = [];
+    // if (phase === 'Running') {
+    //     actionButtons.unshift(
+    //     <Popconfirm
+    //         title='Are you sure you want to cancel this job?'
+    //         onConfirm={(event) => {
+    //             event.stopPropagation();
+    //             // cancelGwasJob("123");
+    //         }}
+    //         okText='Yes'
+    //         cancelText='No'
+    //     >
+    //         <Button type='link' size='medium' danger>cancel job</Button>
+    //     </Popconfirm>);
+    // }
+    if (phase === 'Succeeded') {
+      actionButtons.unshift(
+        <Button
+          primary='true'
+          type='link'
+          size='small'
+          className='GWAS-completedBtn'
+          onClick={(event) => {
+            event.stopPropagation();
+            handleWorkflowOutput(`${gwasWorkflowPath}status/${workflowName}`);
+          }}
+        >
+          download outputs
+        </Button>,
+      );
+    }
+    return actionButtons;
+  };
+
+  const getStatusTag = (phase) => {
```

Comment:
For phase we should also add Canceled and Canceling

Reply:
added statuses for those now